actions/run_projman_filler.yaml

```diff
@@ -0,0 +1,26 @@
+---
+    name: 'run_projman_filler'
+    runner_type: 'remote-shell-cmd'
+    description: 'run projman_filler on remote host'
+    enabled: true
+    entry_point: ''
+    parameters:
+      projman_env_var:
```

Comment:
One suggestion is to rename this to something like `projman_connection_string`, since it is a bit more informative.

Reply:
Good suggestion, will fix!